lib/main_common.pm

```diff
@@ -1655,14 +1655,14 @@ sub load_extra_tests {
 
 sub load_extra_tests_toolkits {
     loadtest "x11/toolkits/prepare";
-    loadtest "x11/toolkits/x11";
-    loadtest "x11/toolkits/tk";
-    loadtest "x11/toolkits/fltk";
-    loadtest "x11/toolkits/motif";
-    loadtest "x11/toolkits/gtk2";
-    loadtest "x11/toolkits/gtk3";
-    loadtest "x11/toolkits/qt4" if is_opensuse;
-    loadtest "x11/toolkits/qt5";
+    #loadtest "x11/toolkits/x11";
```

Comment:
This also looks like it shouldn't be here (Sorry, I didn't notice it before)

Reply:
oops